src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -230,6 +230,15 @@ module sonic-device_metadata {
                                  otherwise the maximum available buffers (which are read from SAI) will be
                                  created, regardless of the CONFIG_DB buffers configuration.";
                 }
+
+                leaf suppress-fib-pending {
+                    type stypes:admin_mode;
+                    default enabled;
```

Comment:
I saw the master PR default is `disabled`. Why we change to enabled here?

Reply:
changed to 'disabled'